generators/pythonv2/codegen/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@fern-api/pythonv2-codegen",
```

Comment:
This is petty much a copy/paste, but replaces `csharp` with `pythonv2`.

Reply:
nice!